src/storage/btree.c

```diff
@@ -16212,12 +16400,10 @@ btree_find_next_index_record_holding_current_helper (THREAD_ENTRY * thread_p, BT
  */
 static int
 btree_apply_key_range_and_filter (THREAD_ENTRY * thread_p, BTREE_SCAN * bts, bool is_iss, bool * is_key_range_satisfied,
-				  bool * is_key_filter_satisfied, bool need_to_check_null)
+				  bool * is_key_filter_satisfied)
 {
   int c;			/* comparison result */
```

Comment:
```suggestion
  DB_VALUE_COMPARE_RESULT c;			/* comparison result */
```

Reply:
This is a discussion outside of this issue, but I think it was intentionally set to int.
Except checking DB_UNK separately
You don't need to worry about DB_SUBSET, DB_NE, DB_SUPERSET, etc. Could it be that the original developer was intentionally trying to think only of negative and positive numbers?

I think you need to think about the code below.
   `c = -c;`

Reply:
I agree that that may have been what the previous developer intended.
But I think the below expression in DB_VALUE_COMPARE_RESULT is also correct.
```
c = -c
```

Reply:
The purpose is to express my opinion, so if you read it, please resolve it.

Reply:
OK, I will fix it.
Thank you.